akka-actor/src/main/scala/akka/dispatch/AbstractDispatcher.scala

```diff
@@ -377,16 +377,8 @@ object ForkJoinExecutorConfigurator {
                                threadFactory: ForkJoinPool.ForkJoinWorkerThreadFactory,
                                unhandledExceptionHandler: Thread.UncaughtExceptionHandler)
     extends ForkJoinPool(parallelism, threadFactory, unhandledExceptionHandler, true) with LoadMetrics {
-    override def execute(r: Runnable): Unit = {
-      if (r eq null) throw new NullPointerException("The Runnable must not be null")
-      val task =
-        if (r.isInstanceOf[ForkJoinTask[_]]) r.asInstanceOf[ForkJoinTask[Any]]
```

Comment:
Not the reuse, but the call to fork(). See: https://github.com/akka/akka/issues/17341#issuecomment-99065785 and also my report: https://github.com/akka/akka/issues/17341#issuecomment-99504739